client.py

```diff
@@ -540,24 +539,26 @@ def start_the_game(network, screen, my_pick):
 
         client.play()
 
-
-    except Exception as e:
+    finally:
         network.close()
-        raise e
 
 
-def choose_team(network, my_pick):
-    server_respond = network.send(my_pick)
-    if server_respond == "ok":
-        return True
+def pick_team(network, my_pick):
+    """
+    Sends team & role pick to server, returns True if the pick was accepted
+    """
+    server_response = network.send(my_pick)
+    return server_response == "ok"
+
+
+def start_game(network, screen, team_selector, role_selector, start_game_menu):
+    my_pick = (my_team, my_role) = team_selector.get_value()[1], role_selector.get_value()[1]
```

Comment:
so the hole purpose of the two equal sings is to clarify that the first item is ``my_team`` and the second is ``my_role``?